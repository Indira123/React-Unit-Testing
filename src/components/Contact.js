import React from "react";

const Contact = (props) => {
  return <span id={props.phone}>{props.phone}</span>;
};

export default Contact;
