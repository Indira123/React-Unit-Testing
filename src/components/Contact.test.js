import React from 'react';
import { shallow } from 'enzyme';
import Contact from './Contact';
const phone = '88888888888';

describe('Contact component renders correctly and snapshot', () => {
    it('App should render correctly', () => {
        const component = shallow(<Contact phone={phone} />);
        expect(component).toMatchSnapshot();
    });
});
describe('Contact component renders correctly without props', () => {
    it('App should render correctly', () => {
        const component = shallow(<Contact />);
        expect(component).toMatchSnapshot();
    });
});