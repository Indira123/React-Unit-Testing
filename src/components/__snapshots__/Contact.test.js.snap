// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders correctly App should render correctly 1`] = `ShallowWrapper {}`;

exports[`Contact component renders correctly and snapshot App should render correctly 1`] = `ShallowWrapper {}`;

exports[`Contact component renders correctly without props App should render correctly 1`] = `ShallowWrapper {}`;
